src/seedu/addressbook/data/person/Contact.java

```diff
@@ -0,0 +1,10 @@
+package seedu.addressbook.data.person;
+
+public class Contact {
+    public String address;
+    public String email;
```

Comment:
Having individual fields kinda defeats the purpose of extracting "common things" out.